examples/tests/foo.spec.js

```diff
@@ -1,12 +1,18 @@
 import { Selector } from 'testcafe';
+import { PublicReportingAPI } from '@reportportal/agent-js-testcafe/build/src/publicReportingAPI';
```

Comment:
You can re-export `PublicReportingAPI` from index file to simplify path for users (as [here](https://github.com/reportportal/examples-js/blob/master/example-nightwatch/RTR-exampleChromeDriver/tests/home.js#L1))